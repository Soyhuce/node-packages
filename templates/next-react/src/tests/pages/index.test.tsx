import React from 'react'
import { render } from '@testing-library/react'

import Index from '@/pages/index'

describe('<App />', () => {
  it('should contains hello world', () => {
    const { container } = render(<Index />)
    expect(container.textContent).toBe('Hello world ðŸ¤ ')
  })

  it('renders the correct markup', () => {
    const { container } = render(<Index />)
    expect(container).toMatchSnapshot()
  })
})
