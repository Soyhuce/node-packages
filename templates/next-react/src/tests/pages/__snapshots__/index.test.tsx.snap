// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the correct markup 1`] = `
<div>
  <div
    class="flex items-center justify-center h-full"
  >
    Hello world ðŸ¤ 
  </div>
</div>
`;
